consensus/src/epoch_manager.rs

```diff
@@ -507,6 +660,28 @@ impl EpochManager {
                     "process_sync_info",
                     p.process_sync_info_msg(*sync_info, peer_id).await
                 ),
+                verified_event @ VerifiedEvent::CommitVote(_) => {
```

Comment:
seems we can unify it with verified_event @VerifiedEvent::CommitVote(_) | VerifiedEvent::CommitDecision(_) ?

Reply:
resolved